File: VB/VB/WinSolution.Module/BO.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.Xpo
Imports DevExpress.ExpressApp
Imports DevExpress.Persistent.Base
Imports DevExpress.Persistent.BaseImpl

Namespace WinSolution.Module
	<DefaultClassOptions, DefaultListViewOptionsAttribute(MasterDetailMode.ListViewOnly)> _
	Public Class ListViewOnly
		Inherits BaseObject
		Public Sub New(ByVal session As Session)
			MyBase.New(session)
		End Sub
	End Class
	<DefaultClassOptions, DefaultListViewOptionsAttribute(MasterDetailMode.ListViewAndDetailView)> _
	Public Class ListViewAndDetailView
		Inherits BaseObject
		Public Sub New(ByVal session As Session)
			MyBase.New(session)
		End Sub
	End Class
End Namespace

File: CS/WinSolution.Module/BO.cs
using System;
using DevExpress.Xpo;
using DevExpress.ExpressApp;
using DevExpress.Persistent.Base;
using DevExpress.Persistent.BaseImpl;

namespace WinSolution.Module {
    [DefaultClassOptions]
    [DefaultListViewOptionsAttribute(MasterDetailMode.ListViewOnly)]
    public class ListViewOnly : BaseObject {
        public ListViewOnly(Session session) : base(session) { }
    }
    [DefaultClassOptions]
    [DefaultListViewOptionsAttribute(MasterDetailMode.ListViewAndDetailView)]
    public class ListViewAndDetailView : BaseObject {
        public ListViewAndDetailView(Session session) : base(session) { }
    }
}
